src/teststatistics.jl

```diff
@@ -101,6 +101,36 @@ function get_tmutilde(LL, inits)
     end
 end
 
+@doc raw"""
+Test statistic for discovery of a positive signal
+q_0 = \tilde{t}_0
+See equation 12 in: https://arxiv.org/pdf/1007.1727.pdf for reference.
+Note that this IS NOT a special case of q_\mu for \mu = 0.
+"""
+function get_q0(LL, inits)
+    tmutilde = get_tmutilde(LL, inits)
+    tmutilde(0)
+end
```

Comment:
```suggestion
get_q0(LL, inits) = get_tmutilde(LL, inits)
```

in thsi case it's the same?

Reply:
But here q_0 can be valid for any value of μ

Reply:
you're right